### DISCUSSION TITLE
1.0 Roadmap?

### DISCUSSION THREAD

**Discussion by jonas-lomholdt** [thomhurst/TUnit#1211] [General]
I was wondering if there's any specific plans or roadmap for a 1.0 release? I do appreciate all the work you are putting in to this, but just wondering. Thanks!

**Comment by thomhurst** [comment]
I'm mainly waiting for VS support to come out of preview. 

I'm currently tweaking the API for data generators slightly too.

I don't really have any plans for new features, unless there's something you think would be useful?

**Reply by jonas-lomholdt** [reply]
That sounds great! No I don't have any particular ideas yet, I'm still at the try-it-out stage, but if or when I get some, I will make sure to leave a suggestion in this repo. Thanks! Have a nice weekend.
